source/patterns/@aws-solutions-constructs/core/test/lambda-helper.test.ts

```diff
@@ -115,64 +115,15 @@ test("test FunctionProps for no envrionment variable when runtime = PYTHON", ()
   expect(stack).toHaveResource(
     "AWS::Lambda::Function",
     {
-      Type: "AWS::Lambda::Function",
```

Comment:
This no longer appears to confirm the stated purpose for the test (no environment variables). I'm concerned that our changes keep the tests passing, but the tests are no longer of value.

Reply:
The test still asserts if the lambda runtime is set as python, for proof see [this](https://github.com/awslabs/aws-solutions-constructs/blob/fix-unit-test-kms/source/patterns/%40aws-solutions-constructs/core/test/lambda-helper.test.ts#L122) which was the original intent of the test... the kms key policy was just the noise, so removing it did not affect the purpose of this assertion test

Reply:
The description of unit test is inaccurate, it's intent is to assert the runtime of lambda is python, will update the description